src/build-scripts/install_homebrew_deps.bash

```diff
@@ -20,7 +20,8 @@ if [[ `which brew` == "" ]] ; then
 fi
 
 
-brew update >/dev/null
+brew uninstall openssl
+#brew update >/dev/null
```

Comment:
Why not updating anymore?

Reply:
The update re-downloads a broken package and the whole thing fails.

We'll go back to updating when the broken packages are fixed. It's a known problem affecting many projects.